app/javascript/components/HomePageContent.jsx

```diff
@@ -34,16 +35,19 @@ const LinkableSearchTabs = function(props) {
   // the queryParams object does not support the more typical hasOwnProperty test
   return (
     <div>
-      <nav className="nav search-links" data-analytics-name="search" role="tablist">
-        <Link to={`${basePath}/app/studies${location.search}`}
-          className={showGenesTab ? '' : 'active'}>
-          <span className="fas fa-book"></span> Search studies
-        </Link>
-        <Link to={`${basePath}/app/genes${location.search}`}
-          className={showGenesTab ? 'active' : ''}>
-          <span className="fas fa-dna"></span> Search genes
-        </Link>
-      </nav>
+        <nav className="nav search-links" data-analytics-name="search" role="tablist">
+          <Link to={`${basePath}/app/studies${location.search}`}
+                className={showGenesTab ? '' : 'active'}>
+            <span className="fas fa-book"></span> Search studies
+          </Link>
+          <Link to={`${basePath}/app/genes${location.search}`}
+                className={showGenesTab ? 'active' : ''}>
+            <span className="fas fa-dna"></span> Search genes
+          </Link>
+          <span id='home-page-bookmark'>
+            <BookmarkManager eagerLoad={true} />
+          </span>
+        </nav>
```

Comment:
Could you de-indent this block to use the standard SCP JS 2-space indent?  That'd denoise the changeset.